backend/models/project.js

```diff
@@ -0,0 +1,89 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+/**
+ * Represents every TSE project ever started since the dawn of time
+ *
+ * Contains all the people who have/are working on the project as well as other
+ * information such as project timeline, relevant files, etc.
+ */
+module.exports = mongoose.model(
+  "Project",
+  new Schema({
+    name: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    client: {
+      type: String,
+      required: true,
+    },
+    phase: {
+      type: String,
+      required: true,
+    },
+    timeline: {
+      start: {
+        quarter: {
+          type: String,
+          required: true,
+        },
+        year: {
+          type: Number,
+          required: true,
+        },
+      },
+      end: {
+        quarter: {
+          type: String,
+          required: false,
+        },
+        year: {
+          type: Number,
+          required: false,
+        },
+      },
+    },
```

Comment:
Do you think this level of granularity is needed for project timeline? I was thinking maybe down the road, sorting/querying projects based on year or quarter specifically might be needed.

Reply:
I see. Yeah, probably not. If it does come to that, I guess we could easily make a custom sort function.